nft-simple/src/internal.rs

```diff
@@ -1,6 +1,7 @@
 use crate::*;
+use std::mem::size_of;
 
-const YOCTO_MULTIPLIER: f32 = 0.000000000000000000000001;
+pub(crate) const YOCTO_MULTIPLIER: f32 = 0.000000000000000000000001;
```

Comment:
Don't use it just for logging. It adds about 30Kb of code

Reply:
Why do we use floats